website/content/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-k8s.mdx

```diff
@@ -0,0 +1,71 @@
+---
+layout: docs
+page_title: Use JWTs to verify requests to API gateways on Kubernetes
+description: Learn how to use JSON web tokens (JWT) to verify requests from external clients to listeners on an API gateway on Kubernetes-orchestrated networks.
+---
+
+# Use JWTs to verify requests to API gateways on Kubernetes
+
+This topic describes how to use JSON web tokens (JWT) to verify requests to API gateways deployed to Kubernetes-orchestrated containers. If your API gateway is deployed to virtual machines, refer to [Use JWTs to verify requests to API gateways on VMs](/consu/docs/connect/gateways/api-gateway/secure-traffic/verify-jwts-vms).
+
+## Overview
+ 
+You can configure API gateways to use JWTs to verify incoming requests so that you can stop unverified traffic at the gateway. You can configure JWT verification at different levels:
+
+- Listener defaults: Define basic defaults that apply to all routes attached to a listener.
+- HTTP route-specific settings: You can define JWT authentication settings for specific HTTP routes. Route-specific JWT settings override default configurations. 
+- Listener overrides: Define override settings that take precedence over default and route-specific configurations. This enables you to set enforceable policies for listeners.
+
+
+Complete the following steps to use JWTs to verify requests:
+
+1. Define a policy that specifies default and override settings for API gateway listeners and attach it to the gateway.   
+1. Define an HTTP route auth filter that specifies route-specific JWT verification settings.
+1. Attach the auth filter to the HTTP route values file. 
+1. Apply the configurations. 
+
+
+## Requirements
+
+- Consul 1.17 or later
```

Comment:
should this include the consul-k8s version as well?

Reply:
Yes, this is a gap I was hoping someone from Eng could fill in. I don't know which version supports this feature.

Reply:
the corresponding consul-k8s version will be 1.3